import_export/templates/admin/import_export/export.html

```diff
@@ -52,7 +52,9 @@
         {% endif %}
         {{ field.errors }}
 
-        {{ field.label_tag }}
+        {% if not field.field.is_selectable_field %}
```

Comment:
Please can you explain why this if block is needed here and on line 66?  thanks

Reply:
First if block is for fields with `is_selectable_field=False`, so non-checkboxes. This way every regular input has label before the field.

And second if block is for all checkboxes to have labels after. Django does not have a built-in way to place where labels go. So had to change the template